deploy/2_deploy_SimpleAccountFactory.ts

```diff
@@ -1,23 +1,22 @@
 import { ethers } from "hardhat";
 import { DeployFunction } from "hardhat-deploy/types";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { entryPoint07Address } from "viem/account-abstraction";
```

Comment:
this name looks very weird wondering why they made this

Reply:
its an address constant that should be the same on all chains